regression/strings-smoke-tests/java_format3/test.desc

```diff
@@ -1,8 +1,9 @@
 CORE
 test.class
---refine-strings --string-max-length 20
+--refine-strings --string-max-length 20 --verbosity 9
```

Comment:
It's so that we can check that no warning is given. With verbosity lower, no warning is present in the output.